bin/qtui-rebind

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+"""
+    Simple script to rewrite Qt ui files to change shortcut bindings.
```

Comment:
This is all about personal preference for styling, so here is my stance on it.

I prefer
```python
"""
    This is a longer
    docstring.
"""
```
over 
```python
"""This is a longer
docstring.
"""
```
because it makes the block of text stand out much more and thus increase recognisability and readability, especially on module level. They are both equivalent after been trimmed.

I would disagree with the comment suggestion because it is a descriptive information about what the script does - generally speaking.